### PR TITLE
Add a grade | Jaime Sandoval

### PR DESCRIPTION
**Pull request by JaimeGSandoval**
sgt-react-add-grade

### CODE REVIEW

File: client/components/grade-form.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    event.target.reset();
+    this.props.createGrade(name, course, grade);
+  }
+
+  render() {
+    return (
+      <div className="col-4">
+        <form action="/api/grades" onSubmit={this.handleSubmit}>
+          <div className="form-group">
+            <InputFields resetForm={this.resetForm} onChange={this.handleChange} />
```
Comment (thebearingedge) [inline client/components/grade-form.jsx]: I don't recommend making a separate component here. Just put the inputs in the form component directly.

File: client/components/grade-form.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      course: ''
+    });
+  }
+
+  handleChange(event) {
+    this.setState({
+      name: event.target.value,
+      grade: event.target.value,
+      course: event.target.value
+    });
```
Comment (thebearingedge) [inline client/components/grade-form.jsx]: When a change event occurs it should only update the state for the specific field. Use multiple change handlers if you need to. It's fine.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Be sure to debug your state changes with the React dev tools.

File: client/components/grade-form.jsx (hunk 1 above)
Comment (JaimeGSandoval) [reply client/components/grade-form.jsx]: Okay, I'll change it to just one component

File: client/components/grade-form.jsx (hunk 2 above)
Comment (JaimeGSandoval) [reply client/components/grade-form.jsx]: Implemented the individual handlers for each input field to set state with

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**